pkg/serviceaccount/tokens_controller.go

```diff
@@ -307,9 +308,17 @@ func (e *TokensController) createSecret(serviceAccount *api.ServiceAccount) erro
 
 	_, err = serviceAccounts.Update(serviceAccount)
```

Comment:
> I'd like to do that check prior to creating the secret, just to save a potential reason for cleanup.

Is it likely to fail at that point?  Seems like it's relatively unlikely to fail, since the time window between notification and before creating the token is much smaller than the delay caused by the token creation itself.